### DISCUSSION TITLE
Show related data on insights panel

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#14169] [Archive]
Hi!

Let's say I have an Orderlines collection. An orderline has:
- Product -> relation to Products collection
- Quantity (number of products ordered)
- Price (the price at which a product is ordered)

Can I create a panel that shows me top selling/grossing products over time? :)

Might be a stretch, but would be awesome!

**Comment by rijkvanzanten** [comment]
"Top selling" in this case would be the output of "Show product from orderlines sorted by quantity * price" right? There's currently no way to do "inline math" like that yet, but that would definitely be a useful addition. Also curious how that ties in with calculated fields https://github.com/directus/directus/discussions/3079

**Reply by keesvanbemmel** [reply]
That would be even better than the thing I head in mind: "Show product from orderlines sorted by quantity"

**Reply by benhaynes** [reply]
Calculated fields is the ideal solution, as they are dynamic. However, basic math operations within flows could also get this generated statically.

**Reply by rijkvanzanten** [reply]
> "Show product from orderlines sorted by quantity"

The API should already be able to handle this, so this might already be possible through the `list` panel (though I'm not 100% if the GUI allows configuring the correct values, like nested sort)
